Source/DataSources/ModelGraphics.js

```diff
@@ -92,6 +95,13 @@ define([
         minimumPixelSize : createPropertyDescriptor('minimumPixelSize'),
 
         /**
+         * Gets or sets the numeric Property specifying the maximum scale
+         * size of a model. This property is used as an upper limit for 
+         * minimumPixelSize.
+         */
+        maximumScale : createPropertyDescriptor('maximumScale'),
```

Comment:
This override the regular `scale` property as well, right?  That is reasonable, but we should explicitly mention it in the doc.